tests/AgDatabaseMove.Integration/TestAgDatabase.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace AgDatabaseMove.Integration
+{
+  using System.Linq;
+  using Xunit;
+  using Fixtures;
+
+  public class TestAgDatabase : IClassFixture<TestAgDatabaseFixture>
+  {
+    private readonly TestAgDatabaseFixture _fixture;
+
+    public TestAgDatabase(TestAgDatabaseFixture testAgDatabaseFixture)
+    {
+      _fixture = testAgDatabaseFixture;
+    }
+
+    [Fact]
+    public void TestLoginsExist()
+    {
+      _fixture._agDatabase.ContainsLogin(_fixture._loginConfig.LoginName);
+      Assert.True(true);
+    }

```

Comment:
`Assert.True(true)` ?  Can you assert ContainsLogin == true?

Reply:
`ContainsLogin` doesn't return anything, it just throws if there's an error https://github.com/factset/AgDatabaseMove/blob/ad13f56971f5fa176ea523c0206de0a43e9a99e8/src/AgDatabase.cs#L245.

I don't think the `Assert` is necessary for the test to pass though, so I can just remove it.